ogr/abstract.py

```diff
@@ -71,6 +129,38 @@ def __str__(self) -> str:
             f"created='{self.created}')"
         )
 
+    def _get_all_comments(self) -> List[IssueComment]:
+        raise NotImplementedError()
+
+    def get_comments(
+        self, filter_regex: str = None, reverse: bool = False, author: str = None
+    ) -> List[IssueComment]:
+        """
+        Get list of issue comments.
+
+        :param pr_id: int
+        :param filter_regex: filter the comments' content with re.search
+        :param reverse: reverse order of comments
+        :param author: filter comments by author
+        :return: [PRComment]
+        """
+        raise NotImplementedError()
+
+    def can_close(self, username: str) -> bool:
+        raise NotImplementedError()
+
+    def comment(self, body: str) -> IssueComment:
+        raise NotImplementedError()
+
+    def close(self) -> "Issue":
+        raise NotImplementedError()
+
+    def get_labels(self) -> List:
+        raise NotImplementedError()
+
+    def add_labels(self, labels: List[str]) -> None:
```

Comment:
We probably need to have some object representation of the labels because the labels can have the desciption and color.